ts/screens/wallet/payment/PickPspScreen.tsx

```diff
@@ -53,26 +53,26 @@ const style = StyleSheet.create({
 
   flexStart: {
     flexDirection: "row",
-    justifyContent: "flex-start"
+    justifyContent: "flex-start",
+    width: 100,
+    height: 50
```

Comment:
The story about allowing users to select a PSP will make sure to use correct widths and heights here (or at least, the correct ratio between the two). 

Will be creating the story asap